cfgov/cfgov/settings/production.py

```diff
@@ -98,12 +97,16 @@
             'MAX_ENTRIES': 10000,
         },
     },
-    'api_cache':{
+    'api_cache': {
         'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
         'LOCATION': 'api_cache_memory',
         'TIMEOUT': 3600,
         'OPTIONS': {
             'MAX_ENTRIES': 1000,
         },
+    },
+    'post_preview': {
```

Comment:
Without a `TIMEOUT` setting this will [default to 5 minutes](https://docs.djangoproject.com/en/1.8/topics/cache/#cache-arguments). Unless this is set longer, will we get much benefit?

Reply:
Nice catch! Updated to `None`.